src/box/tuple.h

```diff
@@ -748,13 +756,15 @@ tuple_is_compressed(struct tuple *tuple)
  * \param format tuple format.
  * \param data tuple data in MsgPack Array format ([field1, field2, ...]).
  * \param end the end of \a data
+ * \param validate whether the new tuple gets validated against format.
  * \retval tuple on success
  * \retval NULL on out of memory
  */
 static inline struct tuple *
-tuple_new(struct tuple_format *format, const char *data, const char *end)
+tuple_new(struct tuple_format *format, const char *data, const char *end,
+	  bool validate)
```

Comment:
Please don't change the signature of such a widely used function. Better introduce a new function instead.

Reply:
Dropped this change.